test-apps/display-test-app/src/frontend/ClassificationsPanel.ts

```diff
@@ -72,50 +66,11 @@ export class ClassificationsPanel extends ToolBarDropDown {
     this._realityModelListDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
     this._modelListDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
     this._propertiesDiv = IModelApp.makeHTMLElement("div", { parent: this._element });
-    this._realityModelPickerMenu = createNestedMenu({

```

Comment:
@pmconne Is this RealityModelPicker used? It is dependent on functionality that is no longer supported and would be simplest if it could be dropped. But if it is necessary I can attempt to make a work around.

Reply:
Yes, it is used to query reality models that intersect the project extents. It is extraordinarily useful for testing, and other applications like Design Review have similar pickers. What is the functionality that has been dropped? Perhaps dropping it is not good.

Reply:
The repo does not use the querying functionality so I looked at removing it from the interface since the plan was to only enforce what is used in the production code. 
@calebmshafer What are your thoughts on how this functionality should be brought back?

Reply:
Are you including apps like Design Review and the shared modules they depend upon in "production code"?

Reply:
Design Review and others are using a different UI component that @MarcBedard8 wrote. If we were to replace this functionality, I'd rather it follow more closely to what he has done for those apps rather than add back to the core interface something unused otherwise...

Reply:
Link to that component please? As I recall, we added queryRealityData (really, its predecessors) specifically for that UI component.

Reply:
I'll follow up internally as the current source is there.

Reply:
Much of the issue is that queryRealityData is separated from the RealityDataClient. @wgoehrig has suggested that this functionality is put into the RealityDataClient implementation so it would be available. Would that be a reasonable fix?

Reply:
That is exactly what I am suggesting.